### PR TITLE
[OAP-1543][oap-data-source][arrow] File descriptor leak in v1

### PR DESCRIPTION
**Pull request by zhztheplayer**
#1543

### CODE REVIEW

File: oap-data-source/arrow/standard/src/test/scala/com/intel/oap/spark/sql/execution/datasources/arrow/ArrowDataSourceTest.scala [hunk 1]
```diff
@@ -234,6 +234,24 @@ class ArrowDataSourceTest extends QueryTest with SharedSparkSession {
...
+          .asInstanceOf[UnixOperatingSystemMXBean]
+          .getOpenFileDescriptorCount
+    }
+
+    val initialFdCount = getFdCount
+    for (_ <- 0 until 100) {
+      verifyParquet(spark.sql("select * from ptab2"))
+    }
+    val fdGrowth = getFdCount - initialFdCount
+    assert(fdGrowth < 100)
```
Comment (zhouyuan) [inline oap-data-source/arrow/standard/src/test/scala/com/intel/oap/spark/sql/execution/datasources/arrow/ArrowDataSourceTest.scala:252]: can we assert(fdGrowth == 0) here?

Comment (zhztheplayer) [reply oap-data-source/arrow/standard/src/test/scala/com/intel/oap/spark/sql/execution/datasources/arrow/ArrowDataSourceTest.scala:252]: Thank Yuan, I am not sure on this. If ManagementFactory provides system level fd count, I was concerned that the test case may be unexpectedly failed if another process happened to reserve fds during running.

**Comment by zhouyuan** [comment]
Hi @xwu99 Travis CI on mlib seems failing on master, could you please take a look?

**Comment by xwu99** [comment]
> Hi @xwu99 Travis CI on mlib seems failing on master, could you please take a look?

The reason it failed is oneAPI released a new beta version and changed the default path, I will submit PR to fix this.

**Comment by zhztheplayer** [comment]
Let's not to treat this CI failure as blocker as relevant jobs were passed. Merging

**Comment by xwu99** [comment]
@zhztheplayer #1551 temp removed mllib CI.

**Pull request merged**